content/news.md

```diff
@@ -1,4 +1,6 @@
 ## Gold Card News
+- <time datetime="2020-09-14">2020-09-14</time> [Proposed updates to Foreign Talent Act (中文)](https://foreigntalentact.ndc.gov.tw/News_Content.aspx?n=88EDAE1D4C3C77D7&s=C788C1DD74906B8A)
```

Comment:
`中文` > `in Chinese`? I don't think everyone can read chinese, even basic ones :p 

Also maybe we can add more information on the link with the `title` tag? Codewise it will look like: 

```
[Proposed updates to Foreign Talent Act (in Chinese)](https://foreigntalentact.ndc.gov.tw/News_Content.aspx?n=88EDAE1D4C3C77D7&s=C788C1DD74906B8A "introduces bla bla and blah")
```

For this result: 
[Proposed updates to Foreign Talent Act (in Chinese)](https://foreigntalentact.ndc.gov.tw/News_Content.aspx?n=88EDAE1D4C3C77D7&s=C788C1DD74906B8A "introduces bla bla and blah")

Reply:
now wondering if people even read that title tag 🤷